Makefile

```diff
@@ -177,6 +123,13 @@ docker-build-and-push: ## Push multi-arch docker image with the manager.
 	docker buildx build --builder kratix-image-builder --push --platform linux/arm64,linux/amd64 -t ${IMG} .
 	docker buildx build --builder kratix-image-builder --push --platform linux/arm64,linux/amd64 -t ${IMG_MIRROR} .
 
+work-creator-docker-build-and-push:
```

Comment:
`build-and-push-work-creator`?

Reply:
i just moved this one up, but a rename might be good